peers/open_request_violation.js

```diff
@@ -58,6 +63,7 @@ module.exports = args => {
     capacity: args.capacity,
     channel_ages: args.channel_ages,
     fee_rates: args.fee_rates,
+    is_trusted_funding: args.is_trusted_funding,
```

Comment:
how about just `trusted_funding` rather than `is_trusted_funding`

Reply:
Yeah. I'll make that change.